android/app/src/main/AndroidManifest.xml

```diff
@@ -87,5 +87,12 @@
                 <action android:name="com.google.firebase.MESSAGING_EVENT" />
             </intent-filter>
         </service>
+
```

Comment:
(IIUC)

I think it makes sense that https://github.com/getsentry/sentry-react-native/issues/1102 would be the issue with `TextCompressionModule#decompress`, and that that report would be accurate about the problem being debug-mode-only.

```
This should immediately mean that we start reporting to Sentry when
an exception propagates uncaught and crashes the app.
```

I wonder if this deserves an asterisk on the word "should", though, just in case we find an uncaught exception not being reported to Sentry in the future? Maybe just a link to https://github.com/zulip/zulip-mobile/pull/4996#issuecomment-916501317.

Reply:
Sure; added a note about that.